Week-23/todo-list/app.js

```diff
@@ -0,0 +1,28 @@
+const { MongoClient, Collection } = require("mongodb");
```

Comment:
Isn’t this an unused import? If you already are using mongoose, you don’t need to have the native mongodb driver installed, but double check this.